src/components/numeral-date/numeral-date.pw.tsx

```diff
@@ -0,0 +1,711 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import NumeralDate, { NumeralDateProps } from ".";
+import {
+  NumeralDateComponent,
+  NumeralDateControlled,
+} from "./components.test-pw";
```

Comment:
You can remove NumeralDateComponent from "./numeral-date-test.stories.tsx" You can also remove line 17 from the 'export default' section of the same file:
`  excludeStories: ["NumeralDateComponent"],`